nlp_profiler/high_level_features/ease_of_reading_check.py

```diff
@@ -44,7 +44,11 @@ def ease_of_reading_score(text: str) -> float:
     if (not isinstance(text, str)) or (len(text.strip()) == 0):
         return NaN
 
-    return float(flesch_reading_ease(text))
+    score = float(flesch_reading_ease(text))
```

Comment:
good fix, please apply code formatting with the help of `black`

Reply:
See this https://www.freecodecamp.org/news/auto-format-your-python-code-with-black/